### PR TITLE
Downgraded CPG version

### LINKED ISSUE #2007: Downgrade CPG version

### PR DESCRIPTION
**Pull request by nulls**
### What's done:
- reverted `da7060ebceb0db318f750bddc4b039933bdd21d4`
- reverted versions of neo4j drivers and set strictly them
- fixed printing error in logs

It closes #2007

### CODE REVIEW

**Review by akuleshov7 — APPROVED** [review]

File: save-demo-cpg/src/main/kotlin/com/saveourtool/save/demo/cpg/SaveDemoCpg.kt [hunk 1]
```diff
@@ -4,17 +4,11 @@ import com.saveourtool.save.demo.cpg.config.ConfigProperties
 import org.springframework.boot.SpringApplication
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.context.properties.EnableConfigurationProperties
-import org.springframework.data.neo4j.repository.config.EnableNeo4jRepositories
-import org.springframework.http.ResponseEntity
-
-typealias StringResponse = ResponseEntity<String>
```
Comment (akuleshov7) [inline save-demo-cpg/src/main/kotlin/com/saveourtool/save/demo/cpg/SaveDemoCpg.kt:10]: This alias is not used?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/CpgView.kt [hunk 2]
```diff
@@ -176,7 +175,7 @@ val cpgView: VFC = VFC {
             val alertStyle = if (cpgResult.logs.isNotEmpty()) {
                 cpgResult.logs.forEach { log ->
                     when {
-                        log.contains("ERROR") || log.startsWith("Exception:") -> p {
+                        log.contains("ERROR") || log.startsWith("Exception:") -> pre {
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/CpgView.kt:178]: Not related with CPG, but ok.
BTW - what is ‘pre’ ? I see it for the first time

File: save-demo-cpg/src/main/kotlin/com/saveourtool/save/demo/cpg/SaveDemoCpg.kt (hunk 1 above)
Comment (nulls) [reply save-demo-cpg/src/main/kotlin/com/saveourtool/save/demo/cpg/SaveDemoCpg.kt:10]: yes, copy-paste from backend

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/CpgView.kt (hunk 2 above)
Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/CpgView.kt:178]: not formatted: now we print exception with end of lines

**Pull request merged**